src/bmark/bmark_solvers.jl

```diff
@@ -37,24 +38,32 @@ if Pkg.installed("BenchmarkProfiles") != nothing
   * `stats`: a dict of statistics such as obtained from `bmark_solvers()`
 
   #### Keyword arguments
-  Any keyword argument accepted by `BenchmarkProfiles.performance_profile()`.
+  * `cost`: a function `cost(::ExecutionStats)` returning a positive cost value. Usually, time or number of function evaluations.
+  * Any keyword argument accepted by `BenchmarkProfiles.performance_profile()`.
 
   #### Return value
   A profile as returned by `performance_profile()`.
   """
-  function profile_solvers(stats :: Dict{Symbol, Array{Int,2}}; kwargs...)
+  function profile_solvers(stats :: Dict{Symbol, Array{ExecutionStats,1}};
+                           cost :: Function = stat->stat.elapsed_time,
```

Comment:
This assumes that all solvers measure execution time. If the user calls, say, IPOPT, how does that work? Perhaps it would be safer to measure the number of objective evals by default?

Reply:
Agreed on measuring evals by default. I'll remove the `*_by_evaluations` then, and add a disclaimer for how to benchmark using elapsed_time.